### ISSUE TITLE
Create a README

### ISSUE THREAD

**Issue by thyeggman** [BoyC/GW2TacO#1]
A README would make it a lot easier for people to be able to contribute!  Things that I can think of that should be included:

- How to set up a dev environment
- How to build
- How to test
- Format of POI XML paths (could be in the POIs directory)

Thanks 😄

**Comment by nate998877** [comment]
I have the format of the .trl files. Which might be useful to document somewhere. 

4 bytes int: trail file version (always 0 currently) 
4 bytes int: map id 
the rest of the file is float triplets for a stream of positions

I've been slowly working on a program for editing/merging existing taco packs. Which has given me some good insight into the structure of the XML if anyone has question about anything there feel free to ask. It's not something that I want to write up, but if there's any questions feel free to ask.

I appreciate BoyC Open Sourcing this. I think I've even asked about it in the past. Regardless it's here now & it's one of it not the best overlays around! Thanks!

**Comment by coderedart** [comment]
> 
> I've been slowly working on a program for editing/merging existing taco packs. Which has given me some good insight into the structure of the XML if anyone has question about anything there feel free to ask. It's not something that I want to write up, but if there's any questions feel free to ask.
>

@nate998877 any plans to opensource the app you are working on? Would like to see the parsing of XML files and how you maintain an internal representation of them . I had to use hex editors to find out the format of trails along with taco source. And realized a good while later that guid of a poi or trail is actually a base64 encoded uuid to save space I guess. Blish people will eventually document the marker format when they get a wiki up

**Comment by nate998877** [comment]
@coderedart 
https://github.com/nate998877/GW2_taco_merger
I hit a brick wall on the project when trying to decide the appropriate way to structure packs. I took a quick break & life hit me like a ton of bricks so I've been distracted with other things hence the lack of progress. The intention was always to open source it, but it's in pretty rough shape at the moment...

**Comment by coderedart** [comment]
@nate998877 yeah. the format is too vague at the moment. until now anything taco did was effectively standard. with multiple overlays, we will really need stricter rules to maintain compatibility of a marker pack between all the overlays. without coordination, we will just end up with marker packs that only work with a particular overlay, or worse, a particular version of an overlay.
